tests/unit/test_zowe_core.py

```diff
@@ -297,3 +383,47 @@ def test_secure_values_loading_exception(self, get_pass_func):
             prof_manager = ProfileManager()
             prof_manager.config_dir = self.custom_dir
             props: dict = prof_manager.load("ssh")
+
+
+class TestValidateConfigJsonClass(unittest.TestCase):
+    """Testing the validate_config_json function"""
+
+    def test_validate_config_json_parameterized(self):
+        """Test validate_config_json with different values"""
+        test_values = [
+            ((matching_config_json, matching_schema_json), True),
+            ((not_matching_config_json, not_matching_schema_json), False),
+        ]
+
+        current_dir = os.getcwd()
+
+        for test_case in test_values:
+            if test_case[1]:
+                with open("config.json", "w") as file:
```

Comment:
Why do you need to create those files temporarily and then delete them? Can't you have a `resources` folder for your tests and add those files there permanently? Their content is not dinamyc, they are statically defined in the python script.

Reply:
Ok, I'll put them in a separate folder and import them from there.